hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/client/TestReplicationConfig.java

```diff
@@ -159,6 +160,35 @@ public void deserializeEC() {
     validate(config, EcCodec.valueOf(codec), data, parity, chunkSize);
   }
 
+  @Test
+  public void testECReplicationConfigGetReplication() {
+    assumeECType();
+    HddsProtos.ECReplicationConfig proto =
+        HddsProtos.ECReplicationConfig.newBuilder().setCodec(codec)
+            .setData(data).setParity(parity).setEcChunkSize(chunkSize).build();
+
+    ReplicationConfig config =
+        ReplicationConfig.fromProto(ReplicationType.EC, null, proto);
+
+    String ecReplicationParamsDelimiter = "-";
```

Comment:
Do you think it would make sense to make the static `EC_REPLICATION_PARAMS_DELIMITER` in `ECReplicationConfig` public and then reference it here? That way, if we ever changed the constant in `ECReplicationConfig`, it would mean the test doesn't need to be changed.

Reply:
Sure It make sense. Let me change that.